project/webapp/templates/partials/thread/thread_body.html

```diff
@@ -76,4 +214,7 @@
         </div>
     </div>
 </div>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js">
```

Comment:
Script imports should be in the document `head` or near the bottom of the top-level template. Also, be considerate when using *any* third-party code, particularly from Google and Facebook, as it may reduce people's privacy, or worse.